pyproject.toml

```diff
@@ -66,7 +66,7 @@ tests = [
 ]
 
 [project.scripts]
-mne_bids_pipeline = "mne_bids_pipeline.run:main"
+mne_bids_pipeline = "mne_bids_pipeline.run:main_cli"
```

Comment:
the new behavior only affects the CLI; if instead one programmatically calls `run.main()`, everything remains unchanged and one gets the full traceback

Reply:
I think in practice nobody will programatically call `run.main()` (and it should be private).

Have you checked to make sure `run_tests.py` with `--debug` for example give reasonable outputs? And with `on_error = 'abort'` this doesn't now print redundant information or anything? If so then I'm okay with this